lib/canada_post/client.rb

```diff
@@ -33,19 +33,23 @@ def registration_token
       Request::Registration.new(@credentials).get_token
     end
 
+    def get_merchant_info(token)
+      Request::Registration.new(@credentials).merchant_info(token)
+    end
+
     def get_artifact(url)
       manifest = Request::Manifest.new(@credentials).get_manifest(url)
       if manifest[:errors].present?
         return {
-          status: false,
-          error: manifest[:errors]
+            status: false,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.